cypress.config.ts

```diff
@@ -0,0 +1,17 @@
+import { defineConfig } from 'cypress';
+
+export default defineConfig({
+  fixturesFolder: 'tests/cypress/fixtures',
+  screenshotsFolder: 'tests/cypress/screenshots',
+  videosFolder: 'tests/cypress/videos',
+  e2e: {
+    // We've imported your old cypress plugins here.
+    // You may want to clean this up later by importing these.
+    setupNodeEvents(on, config) {
+      return require('./tests/cypress/plugins/index.js')(on, config);
+    },
+    baseUrl: 'http://localhost:1668',
+    specPattern: 'tests/cypress/integration/**/*.cy.{js,jsx,ts,tsx}',
+    supportFile: 'tests/cypress/support/index.js',
+  },
```

Comment:
I would add the following flag to the setting this way you get a button in Cypress to run all tests after each other.

```suggestion
        experimentalRunAllSpecs: true,
  },
```

Reply:
done